x-pack/test/apm_api_integration/trial/tests/transactions/latency.ts

```diff
@@ -65,18 +65,25 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             expect(response.status).to.eql(200);
           });
 
-          it('should return the ML job id for anomalies of the selected environment', () => {
+          it('should only return ML results for the currently selected environment', () => {
```

Comment:
Isn't this in contrast to "should return anomaly data from all ML jobs"? Which one is it?

Reply:
This one has a specific environment in the API call. In that case we only return results for the requested environment.